src/ontology/genepio-edit.owl

```diff
@@ -12644,6 +12705,89 @@ AnnotationAssertion(rdfs:label obo:GENEPIO_0100634 "State or Province or Territo
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100900 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100900 "2023-11-28T13:54:45Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100900 "intestinal content")
+SubClassOf(obo:GENEPIO_0100900 obo:GENEPIO_0100898)
+
+# Class: obo:GENEPIO_0100926 (isolated from single source)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100926 "A process in which specimens are obtained from a single sampling event."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100926 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100926 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100926 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100926 "2023-11-29T11:01:23Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100926 "isolated from single source")
+SubClassOf(obo:GENEPIO_0100926 obo:GENEPIO_0100286)
+
+# Class: obo:GENEPIO_0100929 (PONAR grab sampler)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100929 "A specimen collection device that is used to collect sediment samples from hard bottoms."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100929 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100929 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100929 "2023-11-29T14:21:43Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100929 "PONAR grab sampler")
+SubClassOf(obo:GENEPIO_0100929 obo:OBI_0002814)
+
+# Class: obo:GENEPIO_0100930 (soil sample probe)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100930 "A specimen collection device that is used to collect a soil sample from the ground."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100930 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100930 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100930 "2023-11-29T12:15:31Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100930 "soil sample probe")
+SubClassOf(obo:GENEPIO_0100930 obo:OBI_0002814)
+
+# Class: obo:GENEPIO_0100932 (scooping)
```

Comment:
how about "sample scooping".  "Scooping" is a more general term that may show up in food processing etc., e.g. scooping some ice cream.
Rather than "dig out and move ..." perhaps "dig out and store ..." in line with parent class.

Reply:
Yeah right, so we discussed in the last oc/or to have these as "[specimen collection process](http://purl.obolibrary.org/obo/OBI_0000659)" 
(e.g. collecting specimen with scooping) that relates to "scooping". How about that? 
label: specimen collection with scooping
Definition: A specimen collection process that is used to dig out and store an amount of sample using a scoop.